brave/brave-core
# Added position menu entry to sidebar menu model

fix https://github.com/brave/brave-browser/issues/27160


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch Brave and check sidebar is on left
2. Right click to have sidebar context menu
3. Check context menu has `Move to the right` entry
4. Click `Move to the right` and check sidebar is moved on right
5. Right click to have sidebar context menu
6. Check context menu has `Move to the left` entry
